lib/shoryuken/options.rb

```diff
@@ -57,7 +57,10 @@ def worker_registry=(worker_registry)
       end
 
       def polling_strategy(group)
-        options[group].to_h.fetch(:polling_strategy, Polling::WeightedRoundRobin)
+        option_group = group == 'default' ? options : options[:groups][group]
+        strategy = option_group.to_h.fetch(:polling_strategy, Polling::WeightedRoundRobin)
+        strategy = "Shoryuken::Polling::#{strategy}".constantize if strategy.is_a?(String)
```

Comment:
@atyndall `constantize` depends on `ActiveSupport`.

Maybe the simplest way of doing it would be:

```ruby
if strategy == 'StrictPriority'
  Polling::StrictPriority
else
  Polling::WeightedRoundRobin
end
```

WDYT?